package.json

```diff
@@ -8,6 +8,7 @@
     "start:production": "node ./bin/www"
   },
   "dependencies": {
+    "bcrypt": "^5.0.1",
```

Comment:
This dependency is not necessary. Run `npm uninstall bcrypt` and commit the changes.